jasmine/private/junit_reporter.mjs

```diff
@@ -0,0 +1,13 @@
+import { JUnitXmlReporter } from "jasmine-reporters";
```

Comment:
Is it possible to just `require('./junit_reporter.cjs')`? I always forget how mjs/cjs can mix but I think it's possible?

Reply:
I went back on migrating this to ESM since it isn't actually necessary when we use the right `package.json`.